src/datachain/dataset.py

```diff
@@ -112,7 +112,11 @@ def parse(
 
         if is_listing_dataset(dataset_name):
             dependency_type = DatasetDependencyType.STORAGE  # type: ignore[arg-type]
-            dependency_name = listing_uri_from_name(dataset_name)
+            client, _ = Client.parse_url(
+                listing_uri_from_name(dataset_name),
+                None,  # type: ignore[arg-type]
```

Comment:
Can we fix this instead of ignoring?

Reply:
Yes, this is on my plate for long time. I will see if it makes sense to do it in separate PR or this one

Reply:
Fix in separate PR https://github.com/iterative/datachain/pull/435